client/components/domains/register-domain-step/index.jsx

```diff
@@ -949,6 +938,10 @@ class RegisterDomainStep extends React.Component {
 			suggestions = this.props.defaultSuggestions || [];
 		}
 
+		const showTldFilterBar =
+			config.isEnabled( 'domains/kracken-ui/tld-filters' ) ||
+			abtest( 'showDomainTldFilterBar' ) === 'show';
```

Comment:
If we have the `showDomainTldFilterBar` A/B test, do we really need the feature flag? It could be a bit confusing if we merge this, but don't enable the flag on staging and/or production. Especially since the feature flag is checked inside the component as well (but not the A/B test variation). So it ends up confusing - you force the `hide` variation, but the TLD bar is forced on production and wpcalypso. I'd keep it simple and just leave the A/B test one, unless I missed something.

Reply:
Sounds good to me.

Reply:
I actually went ahead and removed the A/B test from this PR. I'll be sure to spin up an A/B test for all the feature changes we're anticipating for the next redesign milestone.